query/aql_context.go

```diff
@@ -405,3 +407,13 @@ type AQLQueryContext struct {
 func (ctx *OOPKContext) IsHLL() bool {
 	return ctx.AggregateType == C.AGGR_HLL
 }
+
+func isAtomicAggType(aggType C.enum_AggregateFunction) bool {
+	return aggType == C.AGGR_SUM_SIGNED || aggType == C.AGGR_SUM_FLOAT;
```

Comment:
how about `AGGR_SUM_UNSIGNED` for `count`?

Reply:
in cudf's map implementation, sum<uint32_t> is not  using atomic operations, therefore the performance will be bad. In some days we will rewrite the hash map and by that time,we can use hash reduction for count

Reply:
i see, can we use uint64 for count if it's hash reduction..?

Reply:
we can use int64, not worth changing the code now, we are gonna rewrite the hash table ourselves anyway